src/App.scss

```diff
@@ -178,3 +178,17 @@
   color:	#6a6e73;
   line-height: 24px;
 }
+
+// mobile fixes
+@media only screen and (max-width: 768px) {
+  .pf-c-dropdown__menu { right: 0; } // position all dropdown contents left in mobile
```

Comment:
We can achieve the same by using `drodpownProps`:

```jsx
        actionsConfig={{
          dropdownProps: { position: 'right' },
          actions: hasWritePermissions
            ? [
                <Link to={routes.sourcesNew.path} key="addSourceButton">
                  <Button variant="primary" id="addSourceButton">
                    {intl.formatMessage({
                      id: 'sources.addSource',
                      defaultMessage: 'Add source',
                    })}
                  </Button>
                </Link>,
              ]
            : [
                <Tooltip
                  content={intl.formatMessage({
                    id: 'sources.notAdminAddButton',
                    defaultMessage:
                      'To add a source, you must be granted write permissions from your Organization Administrator.',
                  })}
                  key="addSourceButton"
                >
                  <span tabIndex="0">
                    <Button variant="primary" isDisabled id="addSourceButton">
                      {intl.formatMessage({
                        id: 'sources.addSource',
                        defaultMessage: 'Add source',
                      })}
                    </Button>
                  </span>
                </Tooltip>,
              ],
        }}
```

Reply:
@rvsia Using drodpownProps doesn't allow us to position it for mobile only.

Reply:
The dropdown is only on mobile, so it's not an issue here.